### PR TITLE
Add examples usages for clonejob.pl

### PR DESCRIPTION
**Pull request by MalloZup**
Add 3 good examples to clonejob.pl when invoked the usage function on command line

### CODE REVIEW

File: script/clone_job.pl [hunk 1]
```diff
@@ -27,6 +27,16 @@ =head1 SYNOPSIS
 
 clone_job.pl [OPTIONS] JOBID [KEY=[VALUE] ...]
 
+
+EXAMPLES:
```
Comment (okurz) [inline script/clone_job.pl]: Please stick to the POD format (even if you might consider it ugly), e.g. see here
http://juerd.nl/site.plp/perlpodtut

Examples actually belong into SYNOPSIS but should be marked as code sections.

**Comment by okurz** [comment]
If the output now really looks like what you showed then the indention is broken

**Comment by MalloZup** [comment]
no , it doesn't look like the coment

**Comment by MalloZup** [comment]
ok i can correct the identation in pod-style, but i think it would be nice to have this.

Since host and from are not really intutitives , an example make this clear

**Comment by okurz** [comment]
sure, I think it is good to add examples, just put them in proper style :-)

**Comment by MalloZup** [comment]
is this format ok for you?

**Comment by okurz** [comment]
LGTM

**Pull request merged**